plugins/mcreator-localization/lang/texts.properties

```
@@ -937,6 +937,7 @@ blockly.block.entity_is_elytra_flying=Is %1 elytra flying
 blockly.block.entity_is_glowing=Is %1 glowing
 blockly.block.entity_is_swimming=Is %1 swimming
 blockly.block.entity_is_tagged_in=Is %1 tagged in entity tags as %2
+blockly.block.entity_islooking_at_entity=Is %1 looking at %2 with area %3 Ignore blocks: %4
```

The capital I looks a bit weird in the middle of the block. Maybe change the text for something like this
```suggestion
blockly.block.entity_islooking_at_entity=Is %1 looking at %2 with area %3 and ignore blocks: %4
```

---

Just replacing capital <kbd>I</kbd> with <kbd>i</kbd> would be good and also follow other blocks, for instance: https://github.com/MCreator/MCreator/blob/b3999efa48ffee52e5119c962401d7f7198324e1/plugins/mcreator-localization/lang/texts.properties#L1011
```suggestion
blockly.block.entity_islooking_at_entity=Is %1 looking at %2 with area %3 ignore blocks: %4
```